### ISSUE TITLE
Windows 11 Folder Style

### ISSUE THREAD

**Issue by Soitora** [DineshSolanki/FoliCon#184]
**Is your feature request related to a problem? Please describe.**
No Windows 11 style icon available

**Describe the solution you'd like**
Adding a 6th icon/overlay style that makes it blend in with normal Windows 11 icons

**Describe alternatives you've considered**
Either finding another program, or switching the style to something else

**Additional context**
For example, this is how it looks for me today using FolderIco, which is very manual work
![image](https://github.com/DineshSolanki/FoliCon/assets/10836780/2e3d50fe-dc8b-497b-880c-782ec354ed8a)

**Comment by Soitora** [comment]
Hey, the Professional mode requires much more manual control, considering the amount of variety
Also while it's pretty similiar I don't feels it's similiar enough

If you don't want to do this I can try and get used to another format, but I'd much prefer if it matched the OS, thank you

**Comment by DineshSolanki** [comment]
for this case, where different users might want a different folder overlay, I started on a plugin system of FoliCon,

https://github.com/DineshSolanki/FoliCon/tree/feature/overlay-plugin

though its currently on pause due to the issue of malicious code execution through FoliCon.

Once the plugin feature is complete, anyone will be able to create different overlays and share to community as well, instead of relying on the fix overlays shipped with FoliCon.

If you or anyone you know would like to contribute to the development of this feature, I would greatly appreciate it.

**Comment by Soitora** [comment]
Maybe this project can help with the base resource: https://github.com/icon11-community/Folder11
Specifically the figma and illustrator: https://github.com/icon11-community/Folder11/tree/main/template

**Comment by DineshSolanki** [comment]
the issue is vertical poster might not scale properly with this, as this will work best with horizontal posters, I'll still see what we can do

**Comment by Soitora** [comment]
Absolutely, Horizontal is the best fit
If going for Vertical support, centering it vertically is the only option (maybe even 2/3rd up), which may work but not always

**Comment by DineshSolanki** [comment]
> Absolutely, Horizontal is the best fit
> If going for Vertical support, centering it vertically is the only option (maybe even 2/3rd up), which may work but not always

Yeah, it might not be a good icon as most posters are verticals, will still try something

**Comment by Soitora** [comment]
Time to learn AI stuff to have it look through the image to find optimal centering point

**Comment by DineshSolanki** [comment]
> Time for me to learn AI/ML stuff to have it look through the image to find optimal centering point (I wish)

Well it would surely help FoliCon as well😄

**Comment by DineshSolanki** [comment]
@Soitora its completed and will be available in next release.
